.circleci/config.yml

```diff
@@ -160,6 +160,66 @@ jobs:
       - run:
           name: Run the oracle-e2e tests
           command: cd e2e-commons && docker-compose run e2e yarn workspace oracle-e2e run native-to-erc
+  ultimate-native-to-erc-ui:
+    machine:
+      image: circleci/classic:latest
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - run: git submodule update --init
+      - run:
+          name: Install redis tools
+          command: sudo apt-get install -y redis-tools
+      - run:
+          name: Install node
+          command: |      
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v10.15.3
+            nvm alias default v10.15.3
+
+            echo 'export NVM_DIR="/opt/circleci/.nvm"' >> $BASH_ENV
+            echo "[ -s \"$NVM_DIR/nvm.sh\" ] && . \"$NVM_DIR/nvm.sh\"" >> $BASH_ENV
+      - run:
+          name: Install Chrome
+          command: |
+            wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+            sudo dpkg -i google-chrome-stable_current_amd64.deb
+      - run:
+          name: Install yarn
+          command: |
+            curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+            echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+            sudo apt-get update && sudo apt-get install yarn
+      - run:
+          name: Install npm packages
+          command: nvm use default; yarn install --frozen-lockfile
+      - run:
+          name: Prepare the infrastructure
+          command: e2e-commons/up.sh deploy native-to-erc
```

Comment:
Do we need the task `ultimate-native-to-erc-oracle` at all? My understanding is that ultimate e2e testing will run oracle+ui+monitor in one test environment as so transactions will be sent initially with UI, then oracle will relay assets and monitor will check that the state of the bridge contracts is OK

Reply:
I have given it a thought and I agree with you.

My initial thinking was to have 3 separate jobs:
1. Deploy `oracle` -> run `oracle-e2e`
2. Deploy `oracle`+`ui` -> run `ui-e2e`
3. Deploy `oracle`+`monitor` -> run `monitor-e2e`

But there is no need for that, plus with 3 types of bridge we would have 9 separate jobs..

I agree with you and I think we should have just one job for one type of bridge that would:
1. Deploy `oracle`+`ui`+`monitor` -> run `ui-e2e`+`oracle-e2e`+`monitor-e2e`

Reply:
I have made the changes to have one `ultimate-native-to-erc` job instead of `ultimate-native-to-erc-oracle` and `ultimate-native-to-erc-ui`, so please review again.

The last step to `Run the oracle-e2e tests` is maybe not necessary since we test the oracle indirectly through `ui-e2e`, I thought to leave it and possibly remove later if need be.

Reply:
Agree that the last step is not needed. The idea of the ultimate tests was to check that everything works properly after deployment through deployment playbooks. So, indirect testing of the bridge oracle during the UI testing is enough for these purposes. We have anyway the oracle tests in the `e2e-oracle` job, don't we?

Reply:
I have removed the last step.

Yes, we still have the `oracle-e2e` job.

Reply:
thanks!